### PR TITLE
1.4.30-M1 changelog

### PR DESCRIPTION
**Pull request by ddolovov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
+# 1.4.30-M1 (Dev 2020)
```
Comment (homuroll) [inline CHANGELOG.md:1]: Dec

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+  * [KT-43597](https://youtrack.jetbrains.com/issue/KT-43597) Xcode 12.2 support
+  * [KT-43276](https://youtrack.jetbrains.com/issue/KT-43276) Add watchos_x64 target
+  * Support value classes
```
Comment (homuroll) [inline CHANGELOG.md:4]: * Significant improvement of compilation time.

Comment (ddolovov) [reply CHANGELOG.md:4]: @homuroll Do we have a KT issue to reference to? Or at least PR/commits?

Comment (sbogolepov) [inline CHANGELOG.md:4]: It might sound confusing for users who expect "real" value classes.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+  * [KT-43198](https://youtrack.jetbrains.com/issue/KT-43198) Init blocks inside of inline classes
+  * [KT-42649](https://youtrack.jetbrains.com/issue/KT-42649) Fix secondary constructors of generic inline classes
+  * [KT-38772](https://youtrack.jetbrains.com/issue/KT-38772) Support non-reified type parameters in typeOf
+  * Untying LLVM
```
Comment (sbogolepov) [inline CHANGELOG.md:8]: It's not about LLVM only. Let's call it "Improved compiler hackability" :)

Comment (ddolovov) [reply CHANGELOG.md:8]: Hackability is a bad word. I would suggest "customization"

**Review by sbogolepov — APPROVED** [review]

**Review by homuroll — APPROVED** [review]

**Pull request closed without merging**